src/main/scala/com/ing/wbaa/airlock/proxy/handler/radosgw/RadosGatewayHandler.scala

```diff
@@ -36,8 +36,7 @@ trait RadosGatewayHandler extends LazyLogging {
         Map(
           "display-name" -> userName.value,
           "access-key" -> awsAccessKey.value,
-          "secret-key" -> awsSecretKey.value,
-          "system" -> "true"
+          "secret-key" -> awsSecretKey.value
```

Comment:
I think we also need to update readme, currently:
Airlock creates new users **with system roles automatically**, but the Airlock NPA must be manually configured as system user to be able to grant admin rights to other users

Reply:
Good point!